### DISCUSSION TITLE
Publish game on steam

### DISCUSSION THREAD

**Discussion by MasterOfTheMojave** [SebLague/Digital-Logic-Sim#85] [Ideas]
Hi, Sebastian and community!

Today I got acquainted with your game/program. I really liked it, because contains an educational and entertaining character. 

It would be very cool if this game was published in steam with the ability to get achievements such as: "Create OR gate", "Create BUFFER", "Create eXclusive OR" and other primitive schemes. Then move on to more complex ones, such as "OR valve with four inlets", "Excluding OR valve with three inputs", "AND valve NOT with five inputs" as described in the task of the book Digital Design and Computer Architecture by David Harris Sarah Harris (this book we read in my university). Create Buffer, seven-segment indicator circuit, RS ‑ trigger, D ‑ trigger, trigger with reset function ... timer 555 (:

![image](https://user-images.githubusercontent.com/33037492/139583076-d8964c15-938f-4c10-8ba4-3737adfdbdca.png)

[source image](https://pikabu.ru/story/kak_zastavit_mashinu_schitat_pri_pomoshchi_tranzistorov_6765172)

Well, the main essence of the game is to create new complex logic circuits based on already created simple logic elements.

I myself usually chase achievements in steam, and they are the goal of the game for me. It would be great if I didn't just kill my time but spent it with benefit and there was motivation.

In my opinion, all achievements must be hidden. This adds more mystery and motivation to create more schemes and solve the game task.

Maybe I did write something wrong (my technical skills on amateur level), please correct and maybe I didn't convince you, well please write you opinion in a comment this post (:

**Comment by kiwi-fruitiwi** [comment]
I imagine making it into an actual game would require a lot more work. There's a free iOS app called Circuitry that's sort of like what you described; there are many small puzzles to build various types of gates, and a sandbox for being creative. I found it after being introduced to making logic gates by Sebastian's project.

**Comment by nofi-sh** [comment]
came here to suggest something similar, just without achievements... i think it'd be nice to launch the existing itch executable from library.

**Reply by BraveCaperCat2** [reply]
You can already do that by adding the game as a non-steam game. It won't be automatically updated by steam, though.

**Reply by nofi-sh** [reply]
thanks for the suggestion, but i was aware of that feature and was just hoping for something more "streamlined"? it would also help getting support on linux systems with proton when users such as myself use package managers
